ckanext/xloader/plugin.py

```diff
@@ -4,6 +4,10 @@
 
 from ckan import plugins
 from ckan.plugins import toolkit
+from ckan.model.domain_object import DomainObjectOperation
+from ckan.model.resource import Resource
+
+import ckanapi
```

Comment:
`toolkit` should already be able to do everything we need.

Reply:
Using `get_action` from toolkit and removed `ckanapi` dependency